src/main/java/mate/academy/springboot/datajpa/dto/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import mate.academy.springboot.datajpa.dto.category.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.category.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper {
+    public CategoryResponseDto toRespDto(Category category) {
```

Comment:
```suggestion
    public CategoryResponseDto toDto(Category category) {
```
please fix in all places